docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -199,6 +199,7 @@ const client = createClient({
     - `"split"` Break the message up into chunks each of which is smaller than the maximum message size.
       Beware that using `"split"` will sacrifice atomicity of changes! Depending on your use case, this may or may not be problematic.
     - `"experimental-fallback-to-http"` Try sending the update over HTTP instead of WebSockets (experimental).
+    - `"throw"` Will throw an error when encountering a message that's too large
```

Comment:
```suggestion
    - `"throw"` Will throw an error when encountering a message that’s too large.
```

Reply:
Can we add a sentence at the end that explains how to handle this? e.g is it a `try/catch` around `createClient`?

Reply:
I ended up going in a different direction, as throw is not easily caught.